### PR TITLE
Added PowerShell and PSConfig steps to remove-a-server-from-a-farm-in-sharepoint-2013.md

### PR DESCRIPTION
**Pull request by dariomws**
https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/issues/1810

### CODE REVIEW

**Comment by MikePlumleyMSFT** [comment]
@TroyStarr can you review?

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointServer/administration/remove-a-server-from-a-farm-in-sharepoint-2013.md [hunk 1]
```diff
@@ -64,6 +64,62 @@ You can remove a web server or an application server from the server farm by uni
...
+  - **securityadmin** fixed server role on the SQL Server instance. 
+    
+  - **db_owner** fixed database role on all databases that are to be updated. 
+    
+  - Administrators group on the server on which you are running the PSConfig.exe tool. 
+    
+    > [!NOTE]
+    > If you do not have permissions, contact your Setup administrator or SQL Server administrator to request permissions. 
+  
+  - From a command prompt, type the following syntax.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointServer/administration/remove-a-server-from-a-farm-in-sharepoint-2013.md]:
```suggestion
   From a command prompt, type the following syntax.
```

File: SharePoint/SharePointServer/administration/remove-a-server-from-a-farm-in-sharepoint-2013.md [hunk 2]
```diff
@@ -64,6 +64,62 @@ You can remove a web server or an application server from the server farm by uni
...
+<a name="RemoveByPowerShell"> </a>
+
+Before you remove a database server from a farm, make sure that you have moved any databases stored on that server to a different database server in your farm. 
+  
+> [!NOTE]
+> Using PowerShell is the recommended way to remove a server from the farm. 
+  
+ **To remove a server from a SharePoint farm by using PowerShell**
+  
+Verify that you have the following memberships:
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointServer/administration/remove-a-server-from-a-farm-in-sharepoint-2013.md]:
```suggestion
1. Verify that you have the following memberships:
```

File: SharePoint/SharePointServer/administration/remove-a-server-from-a-farm-in-sharepoint-2013.md [hunk 3]
```diff
@@ -64,6 +64,62 @@ You can remove a web server or an application server from the server farm by uni
...
+  - **db_owner** fixed database role on all databases that are to be updated. 
+    
+  - Administrators group on the server on which you are running the PowerShell cmdlets.
+    
+    An administrator can use the **Add-SPShellAdmin** cmdlet to grant permissions to use SharePoint Server 2013 cmdlets. 
+    
+    > [!NOTE]
+    > If you do not have permissions, contact your Setup administrator or SQL Server administrator to request permissions. For additional information about PowerShell permissions, see [Add-SPShellAdmin](/powershell/module/sharepoint-server/Add-SPShellAdmin). 
+  
+1. From a PowerShell command prompt, type the following command.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointServer/administration/remove-a-server-from-a-farm-in-sharepoint-2013.md]:
```suggestion
2. From a PowerShell command prompt, type the following command.
```

File: SharePoint/SharePointServer/administration/remove-a-server-from-a-farm-in-sharepoint-2013.md (hunk 2 above)
Comment (dariomws) [reply SharePoint/SharePointServer/administration/remove-a-server-from-a-farm-in-sharepoint-2013.md]: I think it's ok as it's.

File: SharePoint/SharePointServer/administration/remove-a-server-from-a-farm-in-sharepoint-2013.md (hunk 3 above)
Comment (dariomws) [reply SharePoint/SharePointServer/administration/remove-a-server-from-a-farm-in-sharepoint-2013.md]: I think it's ok as it's.

File: SharePoint/SharePointServer/administration/remove-a-server-from-a-farm-in-sharepoint-2013.md (hunk 1 above)
Comment (dariomws) [reply SharePoint/SharePointServer/administration/remove-a-server-from-a-farm-in-sharepoint-2013.md]: It's not PowerShell here, it was ok.

**Pull request merged**